### ISSUE TITLE
Logs not being sent to standard output/error

### ISSUE THREAD

**Issue by mostlyjason** [linuxserver/docker-lychee#19]
Nginx access and error logs and php logs are being sent a custom file location in config/log. This is an unexpected location and will bloat the container volume and space on disk unnecessarily. Docker convention is to redirect these logs to standard output/error.

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

**Comment by thelamer** [comment]
@mostlyjason this image comes from a baseimage variant that is used for many of our containers: 
https://github.com/linuxserver/docker-baseimage-alpine-nginx
We specifically output these logs to the mounted /config directory because in 99% of scenarios people bind mount this to their host they do not use docker volumes. 
If you have a public endpoint or a heavy use container I highly recommend configuring log rotate on a sys level for those files.

**Comment by mostlyjason** [comment]
This practice completely bypasses the standard set by Docker and prevents using the docker logs command, the Docker logging driver and numerous other drivers written by the community. Requiring a host dependency for log rotation is not clearly explained in the readme. It puts the host system at risk of crashing by filling up all available disk space. The statement on the readme `To monitor the logs of the container in realtime: docker logs -f lychee` is also not completely true.

**Comment by aptalca** [comment]
There are docker logs and there are app logs. For most of our images, the docker logs show you the docker processes like if all services started, etc. The app logs are in most cases write to the/config folder. I'm not aware of any standards set by docker when it comes to logging. When we created these images, docker service didn't even have a logrotater or limiter available for itself.

Anyhow, we have our own standards, so container/docker logs go to docker log and app logs go to the config folder. And a good rule of thumb is, if the errors are in the docker log, let us know, if the errors are in the app logs on the config folder, let the app developers know.

Plus, for a bunch of services that are known to aggressively log, and don't have their own  logrotate jobs, we created them (ie. nginx, php, fail2ban, etc.)
